gptel-bedrock.el

```diff
@@ -628,11 +637,12 @@ STREAM - Whether to use streaming responses or not."
            :header nil           ; x-amz-security-token is set in curl-args if needed
            :models (gptel--process-models models)
 	   :model-region model-region
+           :api-key api-key
```

Comment:
I think this just needs to be replaced with `:key api-key` to use the default gptel field. But can you call this `bedrock-api-key` so it's clear what it refers to?

Reply:
@felipeochoa I've refactored this PR based on your comment, and changed the name of this parameter 👍